internal/cli/cmd/sysdump.go

```diff
@@ -55,6 +55,9 @@ func newCmdSysdump() *cobra.Command {
 	cmd.Flags().BoolVar(&sysdumpOptions.Debug,
 		"debug", sysdump.DefaultDebug,
 		"Whether to enable debug logging")
+	cmd.Flags().StringArrayVar(&sysdumpOptions.ExtraLabelSelectors,
+		"extra-label-selector", nil,
```

Comment:
Given that this flag can potentially take multiple selectors, should it be called `extra-label-selectors`?

```suggestion
		"extra-label-selectors", nil,
```

Reply:
Looking a bit closer, it seems we're rather inconsistent with using singular `label` and `labels` in existing option names already.

Reply:
Good idea! Addressed!

Reply:
> Looking a bit closer, it seems we're rather inconsistent with using singular `label` and `labels` in existing option names already.

Only saw this after already pushing the change. 

As you pointed out, `extra-label-selectors` is slightly different than the others, in that it allows multiple selectors, e.g. `--extra-label-selectors k8s-app=kube-dns --extra-label-selectors foo=bar,baz=bat` will select any pods with the `k8s-app=kube-dns` pods OR any pods with the `foo=bar` AND `baz=bat` labels.

We can also consider renaming `extra-label-selectors` to something a bit different to make that more explicit. I don't have a strong opinion here.

Reply:
Sorry, I've edited my additional comment again while you replied. The existing option names seem to stem from `cilium-sysdump`, so I guess we want to keep them as is for backwards compatibility.

As for this PR:

> As you pointed out, extra-label-selectors is slightly different than the others, in that it allows multiple selectors, e.g. --extra-label-selectors k8s-app=kube-dns --extra-label-selectors foo=bar,baz=bat will select any pods with the k8s-app=kube-dns pods OR any pods with the foo=bar AND baz=bat labels.

Thanks for providing that example. Maybe we could extend the PR description and/or commit message to include this?

> We can also consider renaming extra-label-selectors to something a bit different to make that more explicit. I don't have a strong opinion here.

I think `extra-label-selectors` is fine. The name together with the usage message makes the intended purpose pretty clear IMO.

Reply:
Kept the name, but updated the usage message and commit description to be a bit more helpful (hopefully).